httpclient5/src/main/java/org/apache/hc/client5/http/entity/mime/MultipartEntityBuilder.java

```diff
@@ -204,6 +217,20 @@ public MultipartEntityBuilder addBinaryBody(final String name, final InputStream
         return addBinaryBody(name, stream, ContentType.DEFAULT_BINARY, null);
     }
 
+    /**
+     * Returns the fixed default boundary value.
+     */
+    private String getFixedBoundary() {
+        return BOUNDARY_PREFIX + "7k9p2m4x8n5j3q6t1r0vwyzabcdefghi";
+    }
+
+    /**
+     * Generates a random boundary using UUID.
+     */
+    private String getRandomBoundary() {
+        return BOUNDARY_PREFIX + UUID.randomUUID();
```

Comment:
I must admit that the prefix is redundant if you use UUID, no? It just increates the body...

Reply:
The extra bytes are minimal, and the prefix helps with debugging, compatibility, and tracking multipart boundaries